api/lib/domain/services/mail-service.js

```diff
@@ -151,141 +151,64 @@ function sendCertificationResultEmail({
     fromName: `${frTranslations['email-sender-name']['pix-app']} / ${enTranslations['email-sender-name']['pix-app']}`,
     to: email,
     template: mailer.certificationResultTemplateId,
-    variables: templateParams,
+    variables: templateVariables,
   });
 }
 
 function sendResetPasswordDemandEmail({ email, locale, temporaryKey }) {
```

Comment:
Ajouter `FRENCH_FRANCE` comme valeur par défaut pour la locale. Cela évite la ligne l.159.

```suggestion
function sendResetPasswordDemandEmail({ email, locale = FRENCH_FRANCE, temporaryKey }) {
```

(même chose pour les autres fonctions qui ont le même comportement 😛